boardswarm/src/main.rs

```
@@ -745,7 +747,11 @@ impl boardswarm_protocol::boardswarm_server::Boardswarm for Server {
     ) -> Result<tonic::Response<Self::ConsoleStreamOutputStream>, tonic::Status> {
         let inner = request.into_inner();
         if let Some(console) = self.get_console(inner.console) {
-            Ok(tonic::Response::new(console.output_stream().await))
+            let stream = console
+                .output_stream()
+                .await
+                .map_err(|e| tonic::Status::internal(e.to_string()))?;
```

still always returns an internal error I think? Better to implement `From<ConsoleError> for tonic::Status`  similar to e.g. `VolumeError,` then you can just return the error directly.

---

I forgot about this one (I focused on replacing the ConsoleError).
I'll apply your suggestion to convert ConsoleError to tonic::Status.